ext/vscode/src/utils/azureDevCli.ts

```diff
@@ -50,35 +53,60 @@ export function scheduleAzdInstalledCheck(): void {
     setTimeout(async () => {
         const ver = await azdVersionChecker.getValue();
 
-        if (ver === AzdVersionCheckFailure.NotInstalled && !userWarnedAzdMissing) {
+        if (ver === AzdVersionCheckFailure.NotInstalled && !userWarnedAzdMissing && !azdInstallAttempted) {
             userWarnedAzdMissing = true;
             const response = await vscode.window.showWarningMessage(azdNotInstalledMsg(), {}, ...azdNotInstalledUserChoices());
             await response?.callback();
         }
     }, fiveSeconds);
 }
 
+export function onAzdInstallAttempted(): void {
+    azdInstallAttempted = true;
+
+    // Clear the install state so we'll check again at the next command
+    azdVersionChecker.clear();
+}
+
 function createCli(): AzureDevCli {
     const invocation = getAzDevInvocation();
     const builder = CommandLineBuilder.create(invocation[0], ...invocation.slice(1));
     const azDevCliEnv: NodeJS.ProcessEnv = {
         // eslint-disable-next-line @typescript-eslint/naming-convention
         'AZURE_DEV_USER_AGENT': ext.userAgent
     };
+
     if (!vscode.env.isTelemetryEnabled) {
         azDevCliEnv['AZURE_DEV_COLLECT_TELEMETRY'] = "no";
     }
-    const combinedEnv = { 
+
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    let modifiedPath: string = process.env.PATH!;
+
+    // On Unix, the CLI is installed to /usr/bin/local, which is always going to be in the PATH
```

Comment:
```suggestion
    // On Unix, the CLI is installed to /usr/local/bin, which is always going to be in the PATH
```

Reply:
D'oh! I was going to check this before committing but forgot :smile: